llvm/test/CodeGen/SPIRV/struct_opaque_pointers.ll

```diff
@@ -0,0 +1,15 @@
+; RUN: llc -O0 -mtriple=spirv32-unknown-unknown %s -o - | FileCheck %s
```

Comment:
Please also move the test to the `/pointers` directory and change underscores to dashes in the name.